src/codelabs/async-await.md

```diff
@@ -44,13 +59,26 @@ The following example shows the wrong way to use an asynchronous function
 Before running this example, try to spot the issue -- what do you think the
 output will be?
 
-<iframe
-  src="{{site.dartpad-embed}}?id=5c8c7716b6b4284842f15fe079f61e47"
-  style="border: 1px solid lightgrey; margin-top: 10px; margin-bottom: 25px"
-  frameborder="no"
-  height="420"
-  width="100%" >
-</iframe>
+<?code-excerpt "async_await/bin/get_order_sync_bad.dart" remove="Fetching"?>
```

Comment:
Switching to a non-gist source loses the ability to open this example in its own window... but I'm OK with that.

In fact, although I like the ability to copy the code, I find it really annoying that the icon for that covers the code.

Before:  ![image](https://user-images.githubusercontent.com/2164483/69102302-d528ca80-0a16-11ea-8cdf-30cda1dfe2b5.png)    After: ![image](https://user-images.githubusercontent.com/2164483/69102323-de199c00-0a16-11ea-8a0e-207bfebc9a1d.png)

/cc @RedBrogdon @johnpryan

Reply:
This is good feedback - I've filed an issue: https://github.com/dart-lang/dart-pad/issues/1381